server/routes/pdf.js

```diff
@@ -15,7 +16,7 @@ const {
 
 /**
  * @param {object} args
- * @param {any} args.pdfService
+ * @param {PdfService} args.pdfService
```

Comment:
rather than move this over to typescript - elected to just add enhanced typesafety through use of jsdoc